tooltip.js

```diff
@@ -0,0 +1,16 @@
+function InitializeTooltip(data, linkClass) {
+    console.log(data.image)
```

Comment:
maybe not pushing console.log in code to master might be good going forward
```suggestion

```

Reply:
Yup, will remember that from now